content/people/peymant.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Peyman Tehrani
+role: student-phd
+status: student-phd
+yearjoined: 2017
+portrait: peymant.jpg
+badge: peymant-badge.jpg
+interests:
+  - Resource allocation and optimization algorithms 
+  - Wirelless communications and networks
+  - Machine learning and Reinforcement learning
+  - Internet of things and edge computing
+  - Unmanned aerial vehicles
+education:
+  - degree: M.Sc. in Electerical Engineering-Communication Systems
+    institution: University of Tehran, Iran
+    year: 2016
+  - degree: B.Sc. in Electrical Engineering
+    institution: University of Tehran, Iran
+    year: 2013
+email: peymant@uci.edu
+websites:
+    - title: LinkedIn
+     url:www.linkedin.com/in/peyman-tehrani-939195a3/
+    - title: Google Scholar
+     url: https://scholar.google.com/citations?user 1hbKmhYAAAAJ&hl=en
+bibliography:
+  - items:
+    - tehrani2016resource
+    - mili2016energy
+---
+I received my B.S. and M.S. in Electrical Engineering in Communication systems from University of Tehran, Iran. During My Master studies I was a visiting researcher at the information engineering department of University of Padova, Italy, where, I worked on devising novel resource allocation algorithms for full-duplex OFDMA networks. Before joining UCI, I worked as a signal processing and sensor fusion engineer in Sarvin Tech Company, Tehran, in which we were applying signal processing techniques to smart phone's sensors data for using in applications like indoor positioning and smart activity detection. Now I am a Ph.D. student at the Information and Computer Sciences school of University of California Irvine. My research mainly focuses on the using the application of the optimization and machine learning in the Unmanned Aerial vehicles Networks, edge computing and smart grids in order to make them more efficient and reliable.

```

Comment:
The portrait photo still requires cropping. (See my comment after the previous review.)